### DISCUSSION TITLE
The discord link is broken on GitHub Action.

### DISCUSSION THREAD

**Discussion by SuccinctPaul** [badges/shields#8536] [Q&A]
I've to move. However there is a check for links, which has blocked me.

The action report this link, https://img.shields.io/discord/903339070925721652.svg?logo=discord&style=flat-square, with 502 error. But I can view it locally，as blow.

<img width="761" alt="image" src="https://user-images.githubusercontent.com/108982045/196023062-24e9fa05-94ac-4aab-96b9-0356fe1172c8.png">

You can see the more debug details in action.
https://github.com/move-language/move/actions/runs/3226860623/jobs/5280910454

**Comment by chris48s** [comment]
I can't completely debug this for you. As you note, https://img.shields.io/discord/903339070925721652.svg?logo=discord&style=flat-square is working just fine.

A couple of things you might want to look at..
- Was this issue transient, or does it reproduce consistently
- Assuming it is consistent: If you run this link checker tool locally instead of in GH actions, do you get a 502 locally or only in GH actions?

**Reply by SuccinctPaul** [reply]
HI,  the problem is solved . But I don't know how solved, So, I gonna details it more.
> Was this issue transient, or does it reproduce consistently
> Assuming it is consistent: If you run this link checker tool locally instead of in GH actions, do you get a 502 locally or only in GH actions?
For this bug is transient. and the link checker on GH actions get 502.

I've meet this many times.
